scripts/bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+
+# Script to install Nix/Devbox/Direnv
+#
+#
+export DEVBOX_USE_VERSION="0.7.1"
+
+export NIX_SSL_CERT_FILE="$HOME/.local/state/system-certs.pem"
```

Comment:
This variable doesn't seem to ever get added to the user's shell, so it won't actually do anything right

Reply:
the cert file is add to L18 as a parameter passed to `nix-installer` (which will add it to nix.conf)

Reply:
That makes sense, but it could have any name and doesn't need to be exported right?